components/navigation/navigation.css

```diff
@@ -0,0 +1,99 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@200;300;400;500;700;800;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@200;300;400;600;700;800&display=swap');
+
+* {
+    margin : 0;
+    padding : 0;
+    box-sizing: border-box;
+    font-family: 'Nunito Sans', sans-serif;
+}
+
+.content {
+    grid-area: content;
+    padding: 3rem 1.5rem;
+}
+
+.navigation__heading {
+    padding : .5rem;
+}
+
+.navigation__heading-title {
+    font-size: 2.2rem;
+}
+
+.navigation__heading-subtitle{
+    font-size: 1.2rem;
+    font-weight: 300;
+}
+
+.navigation__examples {
+    margin-top: 2rem;
+    padding : .5rem;
+}
+
+.navigation__examples h2{
+    font-weight: 400;
+}
+
+p {
+    font-size: 1.1rem;
```

Comment:
Avoid element grabbing as these will be imported to the users library.

Reply:
ok 👍